README.md

```diff
@@ -4,6 +4,8 @@
 
 **Pencil2D** is an animation/drawing software for Windows, macOS, Linux, and FreeBSD. It lets you create traditional hand-drawn animation (cartoon) using both bitmap and vector graphics. Pencil2D is free and open source.
 
+Here is a video demonstrating the project: <https://www.youtube.com/watch?v=ma52j9B1kEM&t=17s>
```

Comment:
This should probably be put below the link to the website and rephrased to something more explanatory, for example animation showcase 2020.